### PR TITLE
Basic Hebrew Translations

### PR DESCRIPTION
**Pull request by bennypowers**
Here are Hebrew translations for all the basic funcitonalities. More soon, ב״ה P.S. my username is bennyp on habitrpg :D

### CODE REVIEW

File: locales/he/app.json [hunk 1]
```diff
@@ -1,10 +1,411 @@
...
+  "rewards": "פרסים",
+  "Rewards": "פרסים",
+    "gold": "זהב",
+    "silver": "כסף",
+    "newReward": "פרס חדש",
+    "price": "מכיר",
+  "tags" : "תגים",
+    "editTags" : "ערוך תגים",
+    "newTag" : "תג חדש",
+    "clearFilters" : "מחוק מסננות",
```
Comment (bennypowers) [inline locales/he/app.json]: not sure of the usage here

File: locales/he/app.json [hunk 2]
```diff
@@ -1,10 +1,411 @@
...
+
+
+"_commentdrops" : "DROP SYSTEM",
+  "dropsEnabled" : "ירידות מופעלות!",
```
Comment (bennypowers) [inline locales/he/app.json:120]: usage of "ירידות"?

**Comment by lefnire** [comment]
Thanks for the translation @bennyp123! our translations guy is out for a month or so, we'll probably be pulling in all translations post angular migration - we'll keep this in our back pocket

**Comment by bennypowers** [comment]
Oh that's fine I might even be done the rest by then.

שבת שלום!

**Comment by bennypowers** [comment]
anything going on with this? does it need to be reformatted for the angular site?

**Comment by paglias** [comment]
Mostly done but probably app.json will have to be updated because of some changes on how it works.

**Comment by lefnire** [comment]
merged to translations branch, will keep you posted on translations progress. What's your UUID so I can ++ your contrib lvl?

**Comment by bennypowers** [comment]
b2625cdf-0b4f-4189-8f26-441c4ab059d0

**Pull request closed without merging**